runagent-rust/runagent/README.md

```
@@ -1,432 +1,316 @@
-* Built with [Tokio](https://tokio.rs)
-* Uses [Axum](https://github.com/tokio-rs/axum)
-* SQL powered by [SQLx](https://github.com/launchbadge/sqlx)
-* WebSocket support via [tokio-tungstenite](https://github.com/snapview/tokio-tungstenite)
-
----
-
-Need help? Join our [Discord](https://discord.gg/runagent) or check the documentation!
-
+## Need Help?
 
+- Docs: `docs/sdk/rust/` (coming soon) or https://docs.rs/runagent
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)